### PR TITLE
Remove packagingOptions from navigation component gradle files

### PR DESCRIPTION
**Pull request by ancirja-m**
## 📃 Description
ktx artefacts are affected by the packagingOptions from each gradle file

## 🤔 Motivation and Context
Fix for the artefact

## 🧪 How Has This Been Tested?
Manually, with the navigation sample

## ⚒️ Does the PR contain new tests or refactored old ones?
No

## 📷 Screenshots (if appropriate)
Not needed

## 📦 Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Enhancement to a current functionality
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Bug fix (non-breaking change which fixes an issue. Please add issue link here too).

## ✅ Checklist

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] This PR contains new tests that cover the new code.
- [ ] This PR refactor previous tests to cover the new code.
- [ ] This PR is part of a set of PRs that build a bigger feature. If so, please, add here links to previously merged or open PRs.

### CODE REVIEW

File: navigation/navigation-common-ktx/build.gradle [hunk 1]
```diff
@@ -37,10 +37,6 @@ android {
         }
     }
 
-    packagingOptions {
```
Comment (CesarValiente) [inline navigation/navigation-common-ktx/build.gradle:40]: Why this change? was excluding files that were needed? why we needed before/was working before?

Comment (ancirja-m) [reply navigation/navigation-common-ktx/build.gradle:40]: To be honest, I don't remember why I added that on the gradle files, but with that lines the code from the ktx artifacts is not visible.

**Review by CesarValiente — APPROVED** [review]
LGTM! Thanks Andrei!

**Review by bimiron — APPROVED** [review]
